config/environments/test.rb

```diff
@@ -33,7 +33,7 @@
 
   config.action_mailer.perform_caching = false
 
-  config.action_mailer.default_url_options = { :host => "http://localhost:8080" }
+  config.action_mailer.default_url_options = { :host => (Figaro.env.domain_name ? "http://#{Figaro.env.domain_name}" : "http://localhost:8080") }
```

Comment:
we might also need a (non-defaulting-to-localhost) version of this in the production config too!